### PR TITLE
Feature/about

### PR DESCRIPTION
**Pull request by JeremiahPinto**

### CODE REVIEW

File: pages/_document.tsx [hunk 1]
```diff
@@ -29,6 +29,19 @@ export default class MyDocument extends Document {
             integrity='sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf'
             crossOrigin='anonymous'
           />
+          {/* light/dark theme favicons */}
```
Comment (noobling) [inline pages/_document.tsx]: Why is this commented out?

File: pages/_document.tsx [hunk 2]
```diff
@@ -29,6 +29,19 @@ export default class MyDocument extends Document {
...
+            media='(prefers-color-scheme:dark)'
+            href='favicon-dark.png'
+            type='image/png'
+          />
+          <link
+            rel='icon'
+            media='(prefers-color-scheme:light)'
+            href='favicon-light.png'
+            type='image/png'
+          /> */}
```
Comment (noobling) [inline pages/_document.tsx]: Why is this commented out?

File: pages/about.tsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import { Jumbotron, Container, Row, Col } from 'reactstrap'
+import { CommitteeCard } from '../components/Committee'
+import Clients from '../components/Clients'
+import Sponsors from '../components/Sponsors'
+import Partners from '../components/Partners'
+import SpecialThanks from '../components/SpecialThanks'
+
+export default (props: {}) => (
```
Comment (noobling) [inline pages/about.tsx]: Should be able to simply this to `() => <>....</>`

File: pages/index.tsx [hunk 4]
```diff
@@ -4,7 +4,7 @@ import Clients from '../components/Clients'
 import Services from '../components/Services'
 import constants from '../data/constants.json'
 
-export default (props: {}) => (
+export default () => (
```
Comment (noobling) [inline pages/index.tsx]: FYI you don't need those braces `()` its only when you return an object `{}` its required

**Comment by noobling** [comment]
@JeremiahPinto You might want to check why those images aren't rendering

**Comment by Drewbi** [comment]
I just added all the images we have, still need David, Ekin and Thai tho

**Comment by Drewbi** [comment]
Should we change the makers logo to leave out the UWA logo, seems kinda strange and unnecessary

**Comment by noobling** [comment]
> I just added all the images we have, still need David, Ekin and Thai tho.
> They aren't showing up on the deployment yet idk why.

Because its not imported correctly

**Comment by JKKim98** [comment]
yaaay, the images are loading now, albeit very slowly...
btw was it working on local for you guys?

**Comment by Drewbi** [comment]
Yeah, i've tried putting the static directory inside a public directory but it isn't doing anything, not sure if there's an extra config step i'm missing? @noobling

**Comment by Drewbi** [comment]
This isn't working atm but i'm pushing so you can have a look at what i'm doing wrong here

File: pages/_document.tsx (hunk 2 above)
Comment (Drewbi) [reply pages/_document.tsx]: Jerry trying to sneak in that light & dark theme. I don't think it needs it, the entire website is black and white, just invert your screen colours and you done.

Comment (JeremiahPinto) [reply pages/_document.tsx]: Commented out coz we don't have favicon yet

File: components/Committee/CardItem/index.tsx [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+    name,
+    position,
+    about,
+    social,
+    picture: { src, alt }
+  }
+}: Props) => (
+  <Card className='m-3 border-0'>
+    {style}
+    <CardImg top width='100%' src={src} alt={alt} />
```
Comment (Drewbi) [inline components/Committee/CardItem/index.tsx]: I don't think that `top width` does anything, i mean it''s two separate words

**Pull request merged**